packages/react/__tests__/src/components/Alert/index.js

```diff
@@ -27,3 +27,12 @@ test('should return no axe violations', async () => {
   );
   expect(await axe(alert.html())).toHaveNoViolations();
 });
+
+test('should return no axe violations warning variant', async () => {
+  const alert = mount(
+    <Alert show={true} variant={'warning'} heading={'title'}>
```

Comment:
Just a couple of nits:

```suggestion
    <Alert show={true} variant="warning" heading="title">
```

Reply:
Thank you for feedback, updated